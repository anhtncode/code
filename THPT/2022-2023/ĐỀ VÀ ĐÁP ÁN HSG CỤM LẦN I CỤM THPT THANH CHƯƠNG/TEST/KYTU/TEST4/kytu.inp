lkjhabc ASDF Fvd d fdf ffsnd abbc  dd d adbf sff f  fs fss f sfsf sjff sf sfbb fbffs DAAGDG FBFBMF SFKFHSF FFBNGND GGKHSBA  HKDFBSDS FDSF  FFDFSF FSF dasdabcdgdassS fbffb sdsh ACACHDAHDHDSADAHAHDADAHDAHDHADANBFNFHGHdasaafkgggkgrhg fege fbh
