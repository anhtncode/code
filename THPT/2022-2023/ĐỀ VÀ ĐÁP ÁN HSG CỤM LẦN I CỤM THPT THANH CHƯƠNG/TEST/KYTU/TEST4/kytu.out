a:24
b:17
c:5
d:31
e:2
f:47
g:13
h:17
j:2
k:6
l:1
m:1
n:5
r:1
s:28
v:1
